Scripts/AutoSPSourceBuilder.ps1

```diff
@@ -62,6 +62,8 @@
     If this switch is omitted, and no languages have been specified on the command line, languages are skipped entirely.
 .PARAMETER UseExistingLocalXML
     If you want to use a custom or pre-existing AutoSPSourceBuilder.XML file, or simply want to skip downloading the official one on-the-fly, use this switch parameter.
+.PARAMETER Unattend
+    Disables additional user interation so that command can be run in CI/CD pipelines
```

Comment:
Minor changes, "interation" should be **interaction** and replace "command" with "script"